### PR TITLE
HUST_Electrical Load Identification

### PR DESCRIPTION
**Pull request by tsinghua96**
# Summary
- Summarize the main changes or features of your pull request.
- Describe the development status of your pull request.
- List all the related issues of your pull request.

# ToDos
- [] Todo list 1
- [] Todo list 2

# User Manual
Describe about how to test or reproduce the new features or bugs fixed of your pull request.

### CODE REVIEW

File: arc_design_contest/2019/HUST_electrical_load_identification/makefile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+APPL_DEFINES = -DUSE_APPL_MEM_CONFIG -DV2DSP_XY -DMODEL_BIT_DEPTH=16 -g
+
+# optimization Level
+OLEVEL = O0
+
+#
+# root dir of embARC
+#
+EMBARC_ROOT = ../../
```
Comment (IRISZZW) [inline arc_design_contest/2019/HUST_electrical_load_identification/makefile]: project folder is embarc_osp\embarc_applications\arc_design_contest\2019\HUST_electrical_load_identification.
so the EMBARC_ROOT should be ../../../..

File: arc_design_contest/2019/HUST_electrical_load_identification/process.h [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+
+
+extern RESULT Cnn_Net_Classify(void);
+
+
+
+
+#endif /* _PREPROCESS_H_ */
```
Comment (IRISZZW) [inline arc_design_contest/2019/HUST_electrical_load_identification/process.h]: end file with a new line, please all files.

File: arc_design_contest/2019/HUST_electrical_load_identification/makefile (hunk 1 above)
Comment (tsinghua96) [reply arc_design_contest/2019/HUST_electrical_load_identification/makefile]: Modified

File: arc_design_contest/2019/HUST_electrical_load_identification/process.h (hunk 2 above)
Comment (tsinghua96) [reply arc_design_contest/2019/HUST_electrical_load_identification/process.h]: Modified

**Pull request merged**